common/adaptive-bitrate.md

```diff
@@ -0,0 +1,100 @@
+Adaptive bitrate (ABR) refers to features that enable dynamic adjustments to video quality—to optimise for end-user experience under diverse network conditions. ABR ensures that every participant is able to consume the highest possible quality video in conferencing or streaming use-cases, based on their bandwidth constraints.
+
+In addition to network, ABR can also optimise for the right video quality based on the size of the video element. For example, a video call running on active speaker layout has larger video tiles that require higher quality video track. These adjustments can be made dynamically with adaptive bitrate.
+
+Learn about how 100ms enables adaptive bitrate in:
+
+* [Conferencing scenarios](#abr-in-conferencing)
+* [Live streaming scenarios](#abr-in-live-streaming)
+
+## ABR in conferencing
+
+Peers in 100ms rooms can publish multiple video quality levels simultaneously. This is called “simulcast” in 100ms. Peers that consume these video tracks can upgrade or downgrade video quality.
+
+You can enable simulcast on the publishing role's template, and use manual or automatic layer changes on the subscriber's side.
+
+### Publisher-side configuration
+
+Simulcast configuration is opt-in and can be enabled on the role's template. The role's publish video quality determines video quality layers on simulcast. For example, a role configured to publish at 720p can simulcast 180p, 360p and 720p layers.
+
+| Video publish quality | Possible simulcast layers |
+|-----------------------|---------------------------|
+| 180p                  | 180p                      |
+| 240p                  | 240p                      |
+| 360p                  | 180p, 360p                |
+| 480p                  | 240p, 480p                |
+| 720p                  | 180p, 360p, 720p          |
+| 1080p                 | 270p, 540p, 1080p         |
+
+#### Enable via dashboard
+
+_This feature is in limited availability._
```

Comment:
what does this mean?

Reply:
the dashboard toggle is only available for a limited set of accounts. will be generally available in a week. should i frame this differently?